### PR TITLE
Add tag `TUV_CHANGES` for tuv changes

### PR DESCRIPTION
**Pull request by glistening**
libtuv-DCO-1.0-Signed-off-by: Sanggyu Lee sg5.lee@samsung.com

### CODE REVIEW

**Comment by glistening** [comment]
@yichoi I removed author names in tuv change comments as you requested.

File: include/uv-nuttx.h [hunk 1]
```diff
@@ -49,21 +49,20 @@
...
-// @20170407-galpeter: nuttx started to provide ENOTSUP.
+// TUV_CHANGES@20161130: NuttX does not provide ENOTSUP.
+// TUV_CHANGES@20170407: NuttX started to provide ENOTSUP.
 #ifndef ENOTSUP
 #define ENOTSUP       EOPNOTSUPP
 #endif
 
-// @20171130-sanggyu:
-// Not defined macro. Copied from x86-64 linux system header
+// TUV_CHANGES@20171130
```
Comment (yichoi) [inline include/uv-nuttx.h]: :

File: include/uv-nuttx.h [hunk 2]
```diff
@@ -72,7 +71,7 @@
 #define TCP_NODELAY   1
 
 //-----------------------------------------------------------------------------
-// @20171130-sanggyu:
+// TUV_CHANGES@20171130
```
Comment (yichoi) [inline include/uv-nuttx.h]: :

**Comment by yichoi** [comment]
How about add this policy to README.md ?

File: include/uv-nuttx.h (hunk 2 above)
Comment (glistening) [inline include/uv-nuttx.h]: It was my intention. I added ":" for separator for one line comment.
Do you want to add ":" for all tags?

**Comment by glistening** [comment]
@yichoi Currently README.md does not any content, but it says "You can find project details in wiki".
What about appending this policy in [Getting Involved](https://github.com/Samsung/libtuv/wiki#getting-involved) as last bullet.

**Comment by yichoi** [comment]
@glistening Ok, please write at where you think as decent

**Comment by daeyeon** [comment]
LGTM

**Pull request merged**